### ISSUE TITLE
DB Error: Unknown column 'item.shadow' in 'field list'

### ISSUE THREAD

**Issue by rabuzarus** [friendica/friendica#5247]
With https://github.com/friendica/friendica/commit/23aa9dffa061309026f5717a6ed18a387e5d4d23#diff-043fd70296cbf595b3caed1b2dc79248 i get following database error:
```
2018-06-19T21:15:02Z@log5b297255778024.77414774 [NORMAL]:Worker.php:323:execFunction  Process 11735 - Prio 20 - ID 4590911: Notifier ["Notifier","wall-new","169538"] - Process PID: wrk5b2972561f6e09.31162238

2018-06-19T21:15:02Z@wrk5b2972561f6e09.31162238 [NORMAL]:dba.php:514:p  DB Error 1054: Unknown column 'item.shadow' in 'field list'
Item::select, Item::selectFirst, Notifier::execute, Worker::execFunction, Worker::execute, Worker::processQueue
SELECT `item`.`id`, `item`.`uid`, `item`.`parent`, `item`.`uri`, `item`.`parent-uri`, `item`.`thr-parent`, `item`.`guid`, `item`.`contact-id`, `item`.`owner-id`, `item`.`author-id`, `item`.`type`, `item`.`wall`, `item`.`gravity`, `item`.`extid`, `item`.`created`, `item`.`edited`, `item`.`commented`, `item`.`received`, `item`.`changed`, `item`.`title`, `item`.`body`, `item`.`app`, `item`.`verb`, `item`.`object-type`, `item`.`object`, `item`.`target-type`, `item`.`target`, `item`.`postopts`, `item`.`plink`, `item`.`resource-id`, `item`.`event-id`, `item`.`tag`, `item`.`attach`, `item`.`inform`, `item`.`file`, `item`.`location`, `item`.`coord`, `item`.`allow_cid`, `item`.`allow_gid`, `item`.`deny_cid`, `item`.`deny_gid`, `item`.`private`, `item`.`pubmail`, `item`.`moderated`, `item`.`visible`, `item`.`starred`, `item`.`bookmark`, `item`.`unseen`, `item`.`deleted`, `item`.`origin`, `item`.`forum_mode`, `item`.`mention`, `item`.`rendered-hash`, `item`.`rendered-html`, `item`.`global`, `item`.`shadow`, `item`.`content-warning`, `item`.`id` AS `item_id`, `item`.`network`, `author`.`url` AS `author-link`, `author`.`name` AS `author-name`, `author`.`thumb` AS `author-avatar`, `author`.`nick` AS `author-nick`, `author`.`network` AS `author-network`, `owner`.`url` AS `owner-link`, `owner`.`name` AS `owner-name`, `owner`.`thumb` AS `owner-avatar`, `owner`.`nick` AS `owner-nick`, `owner`.`network` AS `owner-network`, `contact`.`url` AS `contact-link`, `contact`.`name` AS `contact-name`, `contact`.`thumb` AS `contact-avatar`, `contact`.`network`, `contact`.`url`, `contact`.`name`, `contact`.`writable`, `contact`.`self`, `contact`.`id` AS `cid`, `contact`.`alias`, `contact`.`uid` AS `contact-uid`, `contact`.`photo`, `contact`.`name-date`, `contact`.`uri-date`, `contact`.`avatar-date`, `contact`.`thumb`, `contact`.`dfrn-id`, `contact`.`network` AS `contact-network`, `parent-item`.`guid` AS `parent-guid`, `parent-item`.`network` AS `parent-network`, `parent-item-author`.`url` AS `parent-author-link`, `parent-item-author`.`name` AS `parent-author-name`, `event`.`created` AS `event-created`, `event`.`edited` AS `event-edited`, `event`.`start` AS `event-start`, `event`.`finish` AS `event-finish`, `event`.`summary` AS `event-summary`, `event`.`desc` AS `event-desc`, `event`.`location` AS `event-location`, `event`.`type` AS `event-type`, `event`.`nofinish` AS `event-nofinish`, `event`.`adjust` AS `event-adjust`, `event`.`ignore` AS `event-ignore`, `event`.`id` AS `event-id`, `sign`.`signed_text`, `sign`.`signature`, `sign`.`signer` FROM `item` STRAIGHT_JOIN `contact` ON `contact`.`id` = `item`.`contact-id` AND NOT `contact`.`blocked` AND ((NOT `contact`.`readonly` AND NOT `contact`.`pending` AND (`contact`.`rel` IN (2, 3))) OR `contact`.`self` OR (`item`.`id` != `item`.`parent`) OR `contact`.`uid` = 0) STRAIGHT_JOIN `contact` AS `author` ON `author`.`id` = `item`.`author-id` AND NOT `author`.`blocked` STRAIGHT_JOIN `contact` AS `owner` ON `owner`.`id` = `item`.`owner-id` AND NOT `owner`.`blocked` LEFT JOIN `user-item` ON `user-item`.`iid` = `item`.`id` AND `user-item`.`uid` = 0 LEFT JOIN `event` ON `event-id` = `event`.`id` LEFT JOIN `sign` ON `sign`.`iid` = `item`.`id` STRAIGHT_JOIN `item` AS `parent-item` ON `parent-item`.`id` = `item`.`parent` STRAIGHT_JOIN `contact` AS `parent-item-author` ON `parent-item-author`.`id` = `parent-item`.`author-id` WHERE (`item`.`id` = '169538' AND `item`.`visible` = '1' AND `item`.`moderated` = '0') LIMIT 1
```

**Comment by annando** [comment]
This is fixed in https://github.com/friendica/friendica/pull/5246

**Comment by MrPetovan** [comment]
Closed via #5246
